SI/Sem.02/Pract.02/Task01.cpp

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+
+int main() {
+	int year = 0;
+
+	std::cout << "Enter a year: ";
+	std::cin >> year;
+
+	if ((year % 400 == 0) || (year % 4 == 0 && year % 100 != 0)) {
+		std::cout << "Entered year is leap.\n";
+	}
```

Comment:
Suggestion: 
} else {

Reply:
I completely agree. Changed the setting in VS to automatically put it there. From now on the "else" will be there :D